ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md

```diff
@@ -0,0 +1,119 @@
+
+
+## Non-goals
+
+* This ADR does not aim at capturing every form or type of persona. It is purposefully generalised, utilising the most frequent persona types as experienced within the Red Hat 'Managed Services' context. The reason is that every organisation calls their personas by different names (and they will inevitably have different role descriptions). Nevertheless, this document hopes to capture the main types in a translatable way.
+
+
+## Current Architecture
+
+* There was some pre-existing art in an internal Red Hat document entitled the "Application Services Primer". That was used as a source for this ADR.
+
+
+## Proposed Architecture
+
+The proposed content is as follows:
+
+***
+
+**Personas related to "Managed Services" (WIP)**
+
+**Purpose**
+To underpin many other Site Reliability Engineering-related topics, we must define the group of "personas" describing  the interaction of that person (or group of people) with an environment supported by SREs.
+
+**Service Owner**
+
+Every service needs an "owner". That is, a persona that has the primary interest in the commercial success of that service. The Service Owner takes responsibility for the success of the service and provides the strategic leadership and business context that is needed by engineers, especially during the difficult times. Without this persona, the service would not exist, and thus all other personas rely on this one. They are often referred to as the 'Champions' of this service.
+
+**Product Owner/Manager**
+
+With the strategic guidance of the Service Owner, this persona (also sometimes referred to as the Product Manager) decides on the priority of work (in this context, that may be categorized as 'feature' work or 'reliability work'). They take their inputs from many sources. Reliability, of course, is a significant input, but it is not the only input. There are many other factors which affect their decision-making. Often, they need to balance the inputs from customers and the business, with the engineering resources that are available and these demands can often be conflicting. It is the task of this role to successfully navigate all the demands, providing a path forward.
+
+Additionally, many services are dependent on other services, which have their own Product Owners. Therefore, this persona also plays a critical role in managing those dependencies, liaising with other Product Owners and Engineering resources to effectively prioritize work. Using agile concepts, this role is analogous to a 'Product Owner'.
+
+**Engineering Manager**
+
+Work needs to be done. The majority of this work is undertaken by Developers, who comprise teams led by an Engineering Manager. The primary role of this persona is to enable those teams to do the (best) work they need to do, as it is prioritized by the Product Owner. This role must also maintain working relationships with peer Engineering managers, especially where co-dependencies, shared goals and natural affinities exist.
```

Comment:
nit: one instance of "Engineering Manager" and one of "Engineering managers".  If we're going with Title Casing for every reference, the second should probably be "Engineering Managers".

Reply:
Agreed @wking . I will make it so.

Reply:
Fixed